getting-started/run-queries-from-console.md

```diff
@@ -12,106 +12,70 @@ As Timescale Cloud is based on PostgreSQL, you can use lots of [different tools]
 connect to your service and interact with your data. To securely manage your date from inside 
 Console, Timescale offers the following Cloud SQL editors:
```

Comment:
I know we call it `Cloud SQL editor` on the pricing page, but I don't know if it's useful on this docs page.

What if we just said it like:
- Within Timescale Console, there are two ways to run SQL queries against your database
- Within `Ops mode`, click on `SQL Editor`, and you can run ad-hoc ephemeral queries
  - This is useful for quick one-off tasks like creating an index on a small table or inspecting `pg_stat_statements`
- `Data mode` is a much richer experience powered by PopSQL where you can write queries with autocomplete, save them in folders, share them, create charts/dashboards, etc.